#! /bin/sh -e
## 06_dautipps.dpatch by Eduard Bloch <blade@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Little patch to extend error information

if [ $# -ne 1 ]; then
    echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
    exit 1
fi
case "$1" in
       -patch) patch -f --no-backup-if-mismatch -p1 < $0;;
       -unpatch) patch -f --no-backup-if-mismatch -R -p1 < $0;;
	*)
		echo >&2 "`basename $0`: script expects -patch|-unpatch as argument"
		exit 1;;
esac

exit 0
				    
@DPATCH@
diff -urNad /tmp/cdrtools-2.0+a25/cdrecord/cdrecord.c cdrtools-2.0+a25/cdrecord/cdrecord.c
--- /tmp/cdrtools-2.0+a25/cdrecord/cdrecord.c	Thu Jan 15 22:28:20 2004
+++ cdrtools-2.0+a25/cdrecord/cdrecord.c	Wed Mar  3 16:02:16 2004
@@ -446,6 +446,9 @@
 			errmsg("%s%sCannot open SCSI driver.\n", errstr, errstr[0]?". ":"");
 			errmsgno(EX_BAD, "For possible targets try 'cdrecord -scanbus'. Make sure you are root.\n");
 			errmsgno(EX_BAD, "For possible transport specifiers try 'cdrecord dev=help'.\n");
+			errmsgno(EX_BAD, "\n");
+			errmsgno(EX_BAD, "For more information, install the cdrtools-doc\n");
+			errmsgno(EX_BAD, "package and read /usr/share/doc/cdrecord/README.ATAPI.setup .\n");
 			exit(EX_BAD);
 	}
 	/*
diff -urNad /tmp/cdrtools-2.0+a25/mkisofs/mkisofs.c cdrtools-2.0+a25/mkisofs/mkisofs.c
--- /tmp/cdrtools-2.0+a25/mkisofs/mkisofs.c	Wed Jan  7 00:23:46 2004
+++ cdrtools-2.0+a25/mkisofs/mkisofs.c	Wed Mar  3 16:01:00 2004
@@ -3010,9 +3010,9 @@
 	}
 	if (goof) {
 #ifdef	USE_LIBSCHILY
-		comerrno(EX_BAD, "Joliet tree sort failed.\n");
+		comerrno(EX_BAD, "Joliet tree sort failed. The -joliet-long switch may help you.\n");
 #else
-		fprintf(stderr, "Joliet tree sort failed.\n");
+		fprintf(stderr, "Joliet tree sort failed. The -joliet-long switch may help you.\n");
 		exit(1);
 #endif
 	}
